ai/Ai_summary_diary.py

```
@@ -0,0 +1,106 @@
+    f"{BASE_URL}/call_logs/events",
+    headers=HEADERS,
+    json={
+        "event": "end",
+        "callType": "OUTGOING",
+        "startedAt": started_at,
+        "rawTranscript": user_only
+    }
+)
+print("call_log 저장:", resp.status_code, resp.text)
+if resp.status_code != 200:
+    exit()
```

Pylint (3.3.7)

[refactor] Consider using 'sys.exit' instead

(R1722)